eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -0,0 +1,105 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $PullRequestNumber,
+  [Parameter(Mandatory=$True)]
+  [string] $BuildId,
+  [Parameter(Mandatory=$True)]
+  [string] $CommitSha,
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactList,
+  [string] $RepoFullName = "",
+  [string] $ArtifactName = "packages",
+  [string] $APIViewUri = "https://apiview.dev/PullRequest/DetectApiChanges"
+)
+
+# Submit API review request and return status whether current revision is approved or pending or failed to create review
+function Submit-Request($filePath)
+{
+    $repoName = $RepoFullName
+    if (!$repoName) {
+        $repoName = "azure/azure-sdk-for-$Language"
+    }
+    $query = [System.Web.HttpUtility]::ParseQueryString('')
+    $query.Add('artifactName', $ArtifactName)
+    $query.Add('buildId', $BuildId)
+    $query.Add('filePath', $filePath)
+    $query.Add('commitSha', $CommitSha)
+    $query.Add('repoName', $repoName)
+    $query.Add('pullRequestNumber', $PullRequestNumber)
+    $uri = [System.UriBuilder]$APIViewUri
+    $uri.query = $query.toString()
+    Write-Host "Request URI: $($uri.Uri.OriginalString)"
+    try
+    {
+        $Response = Invoke-WebRequest -Method 'GET' -Uri $uri.Uri
+        $StatusCode = $Response.StatusCode
+    }
+    catch
+    {
+        Write-Host "Error $StatusCode - Exception details: $($_.Exception.Response)"
+        $StatusCode = $_.Exception.Response.StatusCode
+    }
+
+    return $StatusCode
+}
+
+function Should-Process-Package($pkgPath, $packageName)
+{
+    $pkg = Split-Path -Leaf $pkgPath
+    $configFileDir = Join-Path -Path $ArtifactPath "PackageInfo"
+    $pkgPropPath = Join-Path -Path $configFileDir "$packageName.json"
+    if (-Not (Test-Path $pkgPropPath))
+    {
+        Write-Host " Package property file path $($pkgPropPath) is invalid."
+        return $False
+    }
+    # Get package info from json file created before updating version to daily dev
+    $pkgInfo = Get-Content $pkgPropPath | ConvertFrom-Json
+    Write-Host "SDK Type: $($pkgInfo.SdkType)"
+    return ($pkgInfo.SdkType -eq "client" -and $pkgInfo.IsNewSdk)
```

Comment:
Is this going to be common for all languages? We might want to consider having some potential override that a given language can provide in their language-settings.ps1

Reply:
I have another pending issue that will set a property in Language settings to support overriding this condition and I will make the change here and create API review from scheduled build to just refer that property for a package.

Reply:
I'm fine with this happening in a follow-up.